browser/src/Plugins/PluginManager.ts

```diff
@@ -23,6 +24,11 @@ export class PluginManager implements Oni.IPluginManager {
     private _anonymousPlugin: AnonymousPlugin
     private _pluginsActivated: boolean = false
     private _installer: IPluginInstaller = new YarnPluginInstaller()
+    private _pluginsLoaded = new Event<void>()
```

Comment:
Added this when initially trying to figure out how to get access to the `VCSProvider` however I've left it in as there didn't seem to be away previously to await all plugins being loaded then execute a function